src/overlays/actors/ovl_En_Niw/z_en_niw.c

```diff
@@ -2,14 +2,33 @@
 
```

Comment:
Could you add a little description card at the top of this file? Just something simple like this https://github.com/zeldaret/mm/blob/c3feee1775d101ebb475162b86f9662a6ec1953e/src/overlays/actors/ovl_En_GirlA/z_en_girla.c#L1-L5